multiversx/mx-chain-go
# Mx 15532 fix epoch start bootstrap for sovereign

## Reasoning behind the pull request
- restart a sovereign node does not start from latest block
  
## Proposed changes
- create different bootstrapping phases for sovereign -> remove the metachain and shard epoch start bootstrap computations

## Testing procedure
- stop and restart a sovereign node and it should continue from where it remained
- also works with scripts/testnet `stop.sh` and `sovereignStart.sh` but set `ALWAYS_UPDATE_CONFIGS=0`

## Pre-requisites

Based on the [Contributing Guidelines](https://github.com/multiversx/mx-chain-go/blob/master/.github/CONTRIBUTING.md#branches-management) the PR author and the reviewers must check the following requirements are met:
- was the PR targeted to the correct branch?
- if this is a larger feature that probably needs more than one PR, is there a `feat` branch created?
- if this is a `feat` branch merging, do all satellite projects have a proper tag inside `go.mod`?
